index.html

```diff
@@ -11,59 +11,266 @@
+                                fill="#505F79" />
+                            <path
+                                d="M19.7587 0.394423C19.4253 -0.00424693 18.8587 -0.103914 18.4253 0.195088L16.492 1.52399C15.592 0.560536 14.3253 0.0289756 12.9587 0.0289756C10.2587 0.0289756 8.02532 2.22166 8.02532 4.94591C8.02532 5.34458 8.05865 5.71002 8.15865 6.07547C5.82532 5.57713 4.42532 3.88279 3.89198 0.826316C3.82532 0.427646 3.52532 0.128643 3.12532 0.0289756C2.75865 -0.070692 2.35865 0.0954206 2.12532 0.394423C2.05865 0.460868 -3.44135 8.10205 4.52532 14.5804C3.75865 16.2083 1.45865 17.6037 0.558651 18.0688C0.125318 18.2681 -0.0746825 18.7332 0.0253175 19.1984C0.125318 19.6635 0.525318 19.9625 0.991984 19.9625H2.72532C12.8587 19.9625 17.4587 14.9459 17.5587 3.71667L19.6587 1.72332C20.0587 1.3911 20.092 0.793093 19.7587 0.394423Z"
+                                fill="#505F79" />
+                        </svg>
+                    </div>
+                </div>
+                <div class="resume">
+                    <article class="in"> Get my resume
+                </div>
+            </div>
+
+
+            <div class="container7">
+                <div class="frame12">
+                    <div class="frame7">
+                        <div class="dropdown">
+                            <i class="fa-thin fa-square-chevron-down  small-dot"></i>
+                        </div>
+                        <div class="languages"> Languages</div>
+                    </div>
+                    <div class="language-list">
+                        <div class="frame9">
+                            <div class="js"> Javascript</div>
+                            <i class="elipse1"></i>
+                        </div>
+                        <div class="frame10">
+                            <div class="js"> HTML </div>
+                            <i class="elipse2"></i>
+                        </div>
+                        <div class="frame11">
+                            <div class="js"> CSS </div>
+                            <div class="elipse3"></div>
+                        </div>
+                    </div>
+                </div>
+                <div class="outside">
+                    <div class="frame8">
+                        <div class="vector"></div>
+                        <div class="frameworks"> Frameworks</div>
+                    </div>
+                    <div class="frame13">
+                        <div class="vector">
+                        </div>
+                        <div class="skills"> Skills </div>
+                    </div>
+                </div>
+            </div>
```

Comment:
[OPTIONAL] On the right side of the `Languages`, `Frameworks` and `Skills`, you will see a directional arrow on the figma design. There is also a horizontal line below both the `Frameworks` and `Skills`. Your design is missing all of these as shown below. Kindly :pray: check and fix them.

Your design             |  Figma design
:-------------------------:|:-------------------------:
![Screenshot from 2022-09-29 19-22-39](https://user-images.githubusercontent.com/65233860/193123574-c3e19484-c069-4dad-a66e-ff86206849bd.png)  |  ![Screenshot from 2022-09-29 19-22-16](https://user-images.githubusercontent.com/65233860/193123604-f89d837a-dc69-42bb-9cae-a959cd7c9004.png)